cactus_progressive_config.xml

```diff
@@ -100,13 +156,16 @@
 		diagonalExpansion="20"
 		constraintDiagonalTrim="14" 
 		minimumBlockDegree="2" 
-		minimumIngroupDegree="0"
+		minimumIngroupDegree="1"
```

Comment:
This change is necessary to prevent alignments containing only outgroups when singleCopyOutgroup is disabled. Otherwise outgroup sequence will make it into the ancestor without any evidence for its presence in the ingroups.